test/spinxsk/spinxsk.c

```diff
@@ -683,6 +683,14 @@ AttachXdpProgram(
     } else {
         rule.Match = XDP_MATCH_ALL;
 
+        if (!(RandUlong() % 2)) {
+            rule.Match = RandUlong() % (XDP_MATCH_IP_NEXT_HEADER + 1);
```

Comment:
Another instance where having `XDP_MATCH_COUNT` would be better.

Reply:
Agreed, though the code added directly below covers the case where we forget to update this.

Reply:
See "Jeff Tippet recommended" comment for why this is the way it is.